Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1

```diff
@@ -61,3 +61,168 @@ Describe "Get-GcLogMetric" {
         { Get-GcLogMetric -Metric "non-existent-metric-name" -ErrorAction Stop } | Should Throw "does not exist"
     }
 }
+
+Describe "New-GcLogMetric" {
```

Comment:
Should you add a test that New-GcLogMetric and Get-GcLogMetric return the same value?